container/image.bzl

```diff
@@ -506,16 +506,21 @@ def _impl(
     )
     _assemble_image_digest(ctx, name, container_parts, output_tarball, output_digest)
 
-    # Symlink config file for usage in structure tests
-    ln_path = config_file.path.split("/")[-1]
+    # Copy config file and its sha file for usage in tests
     ctx.actions.run_shell(
         outputs = [output_config],
         inputs = [config_file],
-        command = "ln -s %s %s" % (ln_path, output_config.path),
+        command = "cp %s %s" % (config_file.path, output_config.path),
+    )
+    output_config_digest = ctx.outputs.config_digest
+    ctx.actions.run_shell(
```

Comment:
Actually, this seems to be causing a failure :(:
```
[1A[K(08:51:50) [35mWARNING: [0merrors encountered while analyzing target '//tests/docker/toolchain_container:test-rbe-test-xenial-with-pkgs': it will not be built

(08:51:50) [32mAnalyzing:[0m 613 targets (355 packages loaded, 12039 targets configured)


[1A[K(08:51:50) [31m[1mERROR: [0mfile 'tests/docker/package_managers/ubuntu_gpg_image.json.sha256' is generated by these conflicting actions:

Label: //tests/docker/package_managers:ubuntu_gpg_image

RuleClass: add_apt_key rule

Configuration: d769d4f1658740e0d5d58d8b62919a32f8f9956cb28577a4c785dd8ea1acb1b5

Mnemonic: Action

Action key: ee4f0b4cb4c32de3f26b7455d961b208ac208084ed6be011fedd92a71e84a38b, 0aa9d5a6def5441bff64d39dd00fe787186a9fff26033595220095eb40734680

Progress message: Action tests/docker/package_managers/ubuntu_gpg_image.json.sha256

PrimaryInput: File:[[<execution_root>]bazel-out/k8-fastbuild/bin]tests/docker/package_managers/ubuntu_gpg_image.0.config.sha256, File:[[<execution_root>]bazel-out/k8-fastbuild/bin]tests/docker/package_managers/ubuntu_gpg_image.key.0.config.sha256

PrimaryOutput: File:[[<execution_root>]bazel-out/k8-fastbuild/bin]tests/docker/package_managers/ubuntu_gpg_image.json.sha256
```

looks like the digest file is already generated elsewhere and that action needs to be updated instead of adding a new one here

Reply:
Updated with fixes for duplicates.
Btw should I be able to run something like `bazel test //...`. It doesn't work out of the box for me at least.

Reply:
So we have some nested workspaces used by e2e tests that's probably messes up "..." invocations. What error do you get?

Reply:
For `bazel test //...` I get
```
ERROR: /data/rules_docker/testing/new_pusher_tests/BUILD:17:15: no such package '@verify_new_pusher_image_contents//image': The repository '@verify_new_pusher_image_contents' could not be resolved and referenced by '//testing/new_pusher_tests:new_push_verify_pushed_configs_and_files.image'
ERROR: Analysis of target '//testing/new_pusher_tests:new_push_verify_pushed_configs_and_files.image' failed; build aborted: Analysis failed
```
and if I try `bazel test //tests/...` I get 
```
ERROR: /data/rules_docker/tests/container/BUILD:283:16: in env attribute of container_image_ rule //tests/container:set_env_make_vars: $(ENV_KEY) not defined. Since this rule was created by the macro 'container_image', the error might have been caused by the macro implementation
ERROR: /data/rules_docker/tests/container/BUILD:283:16: in env attribute of container_image_ rule //tests/container:set_env_make_vars: $(ENV_VALUE) not defined. Since this rule was created by the macro 'container_image', the error might have been caused by the macro implementation
```